tests/templates/kuttl/smoke/30-assert.yaml.j2

```diff
@@ -0,0 +1,157 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+timeout: 600
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: hdfs-namenode-default
+spec:
+  template:
+    spec:
+      containers:
+        - name: namenode
+          resources:
+            limits:
+              cpu: "1"
+              memory: 512Mi
+            requests:
+              cpu: 300m
+              memory: 512Mi
+        - name: zkfc
+          resources:
+            limits:
+              cpu: 400m
+              memory: 500Mi
+            requests:
+              cpu: 100m
+              memory: 500Mi
+{% if lookup('env', 'VECTOR_AGGREGATOR') %}
+        - name: vector
+{% endif %}
+      initContainers:
+        - name: format-namenodes
+          resources:
+            limits:
+              cpu: "1"
+              memory: 512Mi
+            requests:
+              cpu: 300m
+              memory: 512Mi
+        - name: format-zookeeper
+          resources:
+            limits:
+              cpu: "1"
+              memory: 512Mi
+            requests:
+              cpu: 300m
+              memory: 512Mi
+status:
+  readyReplicas: 2
+  replicas: 2
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: hdfs-journalnode-default
+spec:
+  template:
+    spec:
+      containers:
+        - name: journalnode
+          resources:
+            limits:
+              cpu: "1"
+              memory: 512Mi
+            requests:
+              cpu: 300m
+              memory: 512Mi
+{% if lookup('env', 'VECTOR_AGGREGATOR') %}
+        - name: vector
+{% endif %}
+status:
+  readyReplicas: 1
+  replicas: 1
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: hdfs-datanode-default
+spec:
+  template:
+    spec:
+      containers:
+        - name: datanode
+          resources:
+            limits:
+              cpu: "1"
+              memory: 512Mi
+            requests:
+              cpu: 400m
+              memory: 512Mi
+{% if lookup('env', 'VECTOR_AGGREGATOR') %}
+        - name: vector
+{% endif %}
+      initContainers:
+        - name: wait-for-namenodes
+          resources:
+            limits:
+              cpu: "1"
+              memory: 512Mi
+            requests:
+              cpu: 400m
+              memory: 512Mi
+status:
+  readyReplicas: {{ test_scenario['values']['number-of-datanodes'] }}
+  replicas: {{ test_scenario['values']['number-of-datanodes'] }}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: hdfs-namenode-default
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: hdfs-datanode-default
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: hdfs-journalnode-default
+{% if test_scenario['values']['datanode-pvcs'] == '2hdd-1ssd' %}
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: hdd-hdfs-datanode-default-0
+status:
+  accessModes:
+    - ReadWriteOnce
+  capacity:
+    storage: 2Gi
```

Comment:
IMHO we should assert on the request (not the status) as this has caused problems in the past (when status was bigger than request)

Reply:
I did not touch this part :)

Reply:
Ah ok ^^ On the other hand journalnodes will have 500MiB going forward and you may run into the same issue on Azure as I did :P Let's see ;)